Logging for server and OneSDK resolver (#33)
superfaceai/one-service: README.md

   git push heroku main
   ```

## Logs

OneService can print structured logs. To allow it use `--logs=<level>` flag:

```shell
oneservice --logs
oneservice --logs=trace
```

> Set optional level specifier to the desired logging level. In order of priority, available levels are:

Reviewer: Nit

```suggestion
Set the optional logging level specifier. Available levels are (ordered by priority) :
```

Author: Fixed

- 'fatal'
- 'error'
- 'warn'
- 'info'
- 'debug'
- 'trace'

> The logging level is a __minimum__ level. For instance if `logger.level` is `'info'` then all `'fatal'`, `'error'`, `'warn'`, and `'info'` logs will be enabled.

Reviewer: Nit

```suggestion
The logging level is the __minimum__ threshold. When set, every higher level will be logged as well. For instance if `logger.level` is `'info'` then all `'fatal'`, `'error'`, `'warn'`, and `'info'` logs will be enabled.
```

Author: Fixed
